### PR TITLE
Console release notes 2022-09-02

### PR DESCRIPTION
**Pull request by seymourisdead**

### CODE REVIEW

**Review by kelvich — DISMISSED** [review]

File: content/docs/release-notes.md [hunk 1]
```diff
@@ -1,6 +1,18 @@
 ---
 title: Release Notes
 ---
+## 2022-09-02 Console release
+
+### What's new
+
+- UI: Pass OAuth app name to the client
```
Comment (antonyc) [inline content/docs/release-notes.md:8]: let's stick to lowercasing here `Pass` -> `pass`.

File: content/docs/release-notes.md [hunk 2]
```diff
@@ -1,6 +1,18 @@
...
+
+### Bug fixes
+
+- UI: fix title on the Sing in page
+
+- UI: fix postgres version.
```
Comment (ololobus) [inline content/docs/release-notes.md:14]:
```suggestion
- UI: change Postgres version on dashboard to 14.5.
```

File: content/docs/release-notes.md [hunk 3]
```diff
@@ -1,6 +1,18 @@
...
 ---
+## 2022-09-02 Console release
+
+### What's new
+
+- UI: Pass OAuth app name to the client
+
+### Bug fixes
+
+- UI: fix title on the Sing in page
```
Comment (antonyc) [inline content/docs/release-notes.md:12]: `Sing` -> `sign`?

**Review by antonyc — CHANGES_REQUESTED** [review]

File: content/docs/release-notes.md (hunk 3 above)
Comment (ololobus) [inline content/docs/release-notes.md:12]:
```suggestion
- UI: fix title on the 'Sign in' page.
```

File: content/docs/release-notes.md (hunk 1 above)
Comment (ololobus) [inline content/docs/release-notes.md:8]:
```suggestion
- UI: pass OAuth app name to the client
```

File: content/docs/release-notes.md [hunk 4]
```diff
@@ -1,6 +1,18 @@
...
+
+- UI: fix postgres version.
+
```
Comment (ololobus) [inline content/docs/release-notes.md:15]:
```suggestion

- Control plane: fix authentication of concurrent proxy connections to the idle compute node.

```

**Review by ololobus — APPROVED** [review]
See comments

**Review by antonyc — APPROVED** [review]

**Pull request merged**